### DISCUSSION TITLE
[Unstable] You can't run while the turret harpoon is hooking a submarine.

### DISCUSSION THREAD

**Discussion by zhu-rengong** [FakeFishGames/Barotrauma#14626] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

As the title said.

### Reproduction steps

1. Fire a coilgun harpoon to hook a submarine
2. Cancle using periscope
3. Run

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Single player

### Version

Other

### -

v1.6.6.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming in the next unstable build.
